diff --git a/core/gcc/Makefile b/core/gcc/Makefile
index 4024066..d5633f0 100644
--- a/core/gcc/Makefile
+++ b/core/gcc/Makefile
@@ -8,9 +8,6 @@ SRW_SRC_GENESIS_DIR = $(SRW_SRC_DIR)/ext/genesis/genesis_july08
 SH_SRC_PARSE_DIR    = $(SRW_SRC_DIR)/ext/auxparse
 SH_SRC_GEN_MATH_DIR = $(SRW_SRC_DIR)/ext/genmath
 
-CC  = cc
-CXX = c++
-
 SRW_SRC_DEF=	-D_GNU_SOURCE -D__USE_XOPEN2K8 -DFFTW_ENABLE_FLOAT -D_GM_WITHOUT_BASE -DSRWLIB_STATIC -DNO_TIMER -DANSI_DECLARATORS -DTRILIBRARY -DLINUX
 CFLAGS=	-O3 -fPIC -I$(SRW_SRC_GEN_DIR) -I$(SRW_SRC_LIB_DIR) -I$(SH_SRC_PARSE_DIR) -I$(SH_SRC_GEN_MATH_DIR) $(SRW_SRC_DEF) 
 
